presto-docs/src/main/sphinx/functions.rst

```diff
@@ -1,9 +1,19 @@
+.. _functions:
```

Comment:
We can link to the documentation pages directly with custom anchor text, so please remove all of these labels.

Reply:
done